tests/mocked_plans/test_base_plan.py

```diff
@@ -32,3 +33,27 @@ async def test_base_plan(_, sample_plans):
     plan = await generate_plan(analysis_results, args)
 
     assert str(plan) == exp_plan
+
+
+@pytest.mark.asyncio
+@patch("cou.utils.nova_compute.get_instance_count", side_effect=[1, 0, 1])
+async def test_base_plan_non_empty(_, sample_plans, caplog):
```

Comment:
I do not see why this test end up here? I think it should be part of unit tests and not mocked-plans, since we are not testing plan here.

The test itself could be here if we drop part for first and last hypervisors, e.g something like  `exp_plan.replace("...", "")`.

Reply:
Should I create a separate config for this test, or is it okay to just keep sharing 'base.yaml' with mocked plans?